src/app.js

```diff
@@ -55,12 +55,28 @@ function MonitorApp ({
     return JSON.parse(value);
   }
 
-  async function getMonitorUpdates() {
-    return await Promise.all(exitnodeIPs.map(async ip => {
-      let d = await getCacheData(`alive-${ip}`);
-      d = d ? d : { error: util.noCheckInMessage(ip) };
-      d.ip = ip;
-      return d;
+  function countActiveRoutesByExitnode() {
```

Comment:
Let me know if you think there's a better name for this function. Happy to change it to whatever